src/mail_room.py

```diff
@@ -25,15 +24,18 @@ def main_controller():
         elif begin is False:
             to_print = create_report()
             if to_print is False:
-                main_controller()
+                pass
             else:
                 print_report(to_print)
                 write_to_file(to_print)
 
 
 def prompt_usr(prompt):
     """Return the user response to a prompt."""
-    usr_answer = input('{} \n>>>>>> '.format(prompt))
+    try:
+        usr_answer = raw_input('{} \n>>>>>> '.format(prompt))
+    except NameError:
+        usr_answer = input('{} \n>>>>>> '.format(prompt))
```

Comment:
This feels awkward to me, having the `NameError` happen here.  I'd feel a bit better about it if it were to happen up at the top of the file just after the imports. Something like this:

```python
try:
    input = raw_input
except NameError:
    pass
```
Then you'd only be causing that error once, instead of every time you ask the user for input.